README.md

```diff
@@ -103,6 +106,33 @@ Flags:
   -h, --help   help for grant-access
 ```
 
+### Onboard
+```
+Onboard everything necesary for a new application into the Operate-first environment.
+
+Creates the necesary groups, namespaces and roles, for a new project from a deployment file. Has been set up with git ops from the Operate-first/support repository, so one can raise an Onboarding-to-cluster issue and onboard that way.
+
+The onboard command expects an onboarding file, which is setup to work by default with the webhook in operate-first/support. This webhook passes in some necesary and optional parameters:
+  Necesary:
+    - Team Name
+    - Desired Project Names / Namespaces
+    - Project Description
+    - Namespace Quota
+  Optional:
+    - Target Cluster (environment)
+    - Custom Quota
+    - GPG Keys
+For more information on this, see the onboarding template in operate-first/support docs: https://github.com/operate-first/support/blob/main/.github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml
+
+Usage:
+  opfcli onboard <file-path>
+
+Flags:
+  -d, --display-name    provide a shorter name to use to refer to the project in openshift
+  -n, --no-limitrange   Do not set a resource limitrange on this project
+
+```
+
```

Comment:
Please document the format of the onboarding file in this repository (either in the README or as a separate document, if you think it's necessary).